portal/api/portal.go

```diff
@@ -97,9 +98,18 @@ func serveHTTP(cfg *config.Config) (*http.Server, error) {
 		cfg.PrivateKey = path
 	}
 
+	redirect := cfg.SSO.Redirect
+	// Trim sso redirect so that tailing "/" will be removed
+	for len(redirect) > 0 {
+		if strings.HasSuffix(redirect, "/") {
```

Comment:
Prefer to `strings.TrimSuffix(redirect, "/")`.

Reply:
Done!